### PR TITLE
Applicant Timeline - Congrats page

### LINKED ISSUE #4150: Feature - Applicant - Thanks you page when submitting application
# Description
A final thank you page that prompts the user with next steps.

Mostly static text, but dynamic content includes:
- the current job's job title
- the current job's department
- the current job's security level
- links to the relevant security forms
- the current job's submission deadline

Links on this page should be routing the user to their respective versions of pages (e.g. profile, my applications, etc.)

# Images

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #4150 

### Notes:
- [x] Add translations

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -51,27 +51,6 @@ function (): void {
...
-            Route::view('application-06', 'applicant/application/06-skill')->middleware('localOnly')->name('app6');
-            /* Application (Questions) */
-            Route::view('application-07', 'applicant/application/07-questions')->middleware('localOnly')->name('app7');
-            /* Application (Review) */
-            Route::view('application-08', 'applicant/application/08-review')->middleware('localOnly')->name('app8');
-            /* Application (Signature & Submission) */
-            Route::view('application-09', 'applicant/application/09-submit')->middleware('localOnly')->name('app9');
-            /* Application (Congrats) */
-            Route::view('application-10', 'applicant/application/10-congrats')->middleware('localOnly')->name('app10');
-
```
Comment (tristan-orourke) [inline routes/web.php:74]: Don't delete these demo routes yet - they're still useful for now.

Comment (yonikid15) [reply routes/web.php:74]: Those are all duplicates actually haha. I accidentally added them in when fixing conflicts.

Comment (tristan-orourke) [reply routes/web.php:74]: Oh! haha

**Review by tristan-orourke — APPROVED** [review]
Just waiting on translations.

**Review by tristan-orourke — APPROVED** [review]

**Pull request merged**